bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -1335,6 +1335,14 @@ synchronized void updateLastConfirmed(long lac, long len) {
      */
 
     public void asyncReadLastConfirmed(final ReadLastConfirmedCallback cb, final Object ctx) {
+        if (clientCtx.getConf().useExplicitLacForReads) {
+            asyncReadExplicitLastConfirmed(cb, ctx);
+        } else {
+            asyncReadLastConfirmedInternal(cb, ctx);
+        }
+    }
+
+    private void asyncReadLastConfirmedInternal(final ReadLastConfirmedCallback cb, final Object ctx) {
```

Comment:
asyncReadLastConfirmedInternal is a bad name. I would prefer calling it `asyncReadPiggybackLastConfirmed` (in contrast to `asyncReadExplicitLastConfirmed`.

Reply:
Okay